dataactvalidator/config/awardFieldsRaw.csv

```diff
@@ -12,8 +12,8 @@ AwardingOfficeName,awarding_office_name,ASP: Derived,string,100,
 AwardingSubTierAgencyCode,awarding_sub_tier_agency_c,ASP: Required,string,4,"D1, D6"
 AwardingSubTierAgencyName,awarding_sub_tier_agency_n,ASP: Derived,string,100,
 AwardModificationAmendmentNumber,award_modification_amendme,ASP: Optional,string,25,D6
-BusinessFundsIndicator,business_funds_indicator,ASP: Required,string,3,D1
-BusinessTypes,business_types,ASP: Required,string,3,D1
+BusinessFundsIndicator,business_funds_indicator,ASP: Optional,string,3,D1
```

Comment:
Yes, I'll add an item to the validation questions sheet so we can discuss them in the weekly validation meeting